media_driver/linux/gen12/ddi/media_libva_caps_g12.cpp

```diff
@@ -2026,6 +2088,16 @@ VAStatus MediaLibvaCapsG12::CreateEncAttributes(
         attrib.type = (VAConfigAttribType) VAConfigAttribPredictionDirection;
         attrib.value = VA_PREDICTION_DIRECTION_PREVIOUS | VA_PREDICTION_DIRECTION_FUTURE | VA_PREDICTION_DIRECTION_BI_NOT_EMPTY;
         (*attribList)[attrib.type] = attrib.value;
+#if VA_CHECK_VERSION(1, 14, 0)
```

Comment:
1,12,0

Reply:
Should be 1.13.0? VAConfigAttribEncHEVCFeatures/VAConfigAttribEncHEVCBlockSizes were added after 1.12.0 release.

Reply:
@XinfengZhang  I thought feiwan1 is correct, version change is in Thu Apr 1 10:52:38 2021 -0400 456a7be libva 2.12.0.pre1

Reply:
https://github.com/intel/libva/blob/master/NEWS#L13
https://github.com/intel/libva/releases/tag/2.12.0

Reply:
Sync with Carl, use 2.12.0